### PR TITLE
Changes made in af_exp_br

### PR DESCRIPTION
**Pull request by starkaf**

### CODE REVIEW

File: docker/Makefile
Comment (alxhoff) [inline docker/Makefile:1]: is it always usbmon4?

File: utils/usb/__init__.py [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
 import sys
 #from main import log
-from ..model_lab.logger import log
+from ..splitter.logger import log
 
 def init_usbmon() -> bool:
     import os
     dirs = os.listdir("/dev/")
-    if "usbmon0" in dirs:
+    if "usbmon4" in dirs:
```
Comment (alxhoff) [inline utils/usb/__init__.py:10]: should this maybe be passed as an argument in case it varies across systems?

File: utils/usb/debug.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def capture_stream(signalsQ:Queue, dataQ:Queue, timeout:int, l:Log) -> None:
 
     packets = []
     context = StreamContext()
-    capture = pyshark.LiveCapture(interface='usbmon0', display_filter=get_filter(addr))
+    capture = pyshark.LiveCapture(interface='usbmon4', display_filter=get_filter(addr))
```
Comment (alxhoff) [inline utils/usb/debug.py:81]: same here

File: docker/Makefile
Comment (starkaf) [reply docker/Makefile:1]: This is rather random, the usbmon needs to be activated using (sudo modprobe usbmon) and then, using wireshark, I look for the usbmon interface that is monitoring the tpu traffic.

File: utils/usb/__init__.py (hunk 1 above)
Comment (starkaf) [reply utils/usb/__init__.py:10]: yes, would be much better

Comment (starkaf) [reply utils/usb/__init__.py:10]: it varies also across different runs on the same system. if the system reboots, it will surely change.

**Pull request merged**